src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -52,6 +64,53 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
           role: "rowheader",
           "aria-label": "Type",
         },
+        {
+          content: (
+            <div>
+              {device.type == "disk" ? (
+                isRootDisk(device as FormDevice) ? (
+                  <>
+                    <strong>Pool: </strong>
```

Comment:
as discussed, we can use the `ResourceLink` component to link to the storage pool here. The `type` prop for the `ResourceLink` component determines the icon to use